source/includes/_basic_example.md

```diff
@@ -0,0 +1,20 @@
+# Basic Example
+A basic example of test taking flow.
+
+1. Get the Auth token to authenticate the user using the [Get Auth Token](#authentication) API.
+2. Use the token as Authorization header in the following requests.
+3. Now try to get list of exams using the [Get Available Exams](#get-available-exams) API.
+4. Get exam details of an exam in the list of exams using the [Get Exam Details](#get-a-single-exam) API.
+5. Start a new attempt for the exam using the [Create an attempt](#create-an-attempt) API.
+6. Using the response of the new attempt, get the questions for that attempt using the [Get attempt questions](#get-attempt-questions) API.
+7. Attend the questions by updating the selected answers or review later flag using [Update attempt questions](#update-attempt-questions) API.
+8. Send heart beat every 1 minute using [Send Heart Beat](#send-heart-beat) API.
+9. [End the attempt](#end-an-attempt) on completed answering the questions.
```

Comment:
~on completed answering the questions~ once all the required questions have been attempted.